pkg/okta/okta.go

```diff
@@ -13,7 +13,11 @@ import (
 	"github.com/cmsgov/easi-app/pkg/apperrors"
 	"github.com/cmsgov/easi-app/pkg/authn"
 	"github.com/cmsgov/easi-app/pkg/handlers"
-	"github.com/cmsgov/easi-app/pkg/models"
+)
+
+const (
+	grtStandard = "EASI_P_GOVTEAM"
```

Comment:
This might be better named grtStandardJobCode

Reply:
Updating these names